CanonicalLayouts/supporting-pane-views/app/src/main/java/com/google/supporting/pane/views/Data.kt

```diff
@@ -23,4 +23,6 @@ val data = mapOf(
     "backend" to listOf("kotlin", "java"),
     "java" to listOf("backend", "android", "desktop"),
     "flutter" to listOf("android", "desktop")
-)
\ No newline at end of file
+)
+
+data class State(val key: String = data.keys.first(), val items: List<String> = data.values.first())
```

Comment:
Nit: it feels a bit weird that there are default parameters here, I wouldn't expect a `State` data object like this one to have them and pulling from static resources. (Granted that this is static data which is unrealistic in the first plac, doesn't matter too much.